core/scss/utilities/mixins/cta/_cta--button.scss

```diff
@@ -32,6 +25,14 @@
     }
   }
 
+  .su-cta__img {
+    @include image-defaults;
+    width: 100%;
+    z-index: 0;
+    will-change: transform;
```

Comment:
This property is new to me. Thanks for the learning opportunity. At my first reading of the spec on https://developer.mozilla.org/en-US/docs/Web/CSS/will-change it indicates that this option should be used very sparingly. Could you add an SCSS comment to what you are trying to fix with this so I can better understand its need?

Reply:
Actually you're right - thank you for making me think about this more carefully. I learned about this property while looking at the Stanford homepage code and thought maybe it would make the animation a bit smoother, but I don't think it's doing much anyway and probably should be used only when really needed. It's now removed.

Reply:
Ok great.